hs_core/tests/api/rest/test_resource_list.py

```diff
@@ -340,3 +340,34 @@ def test_resource_list_by_user(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         content = json.loads(response.content.decode())
         self.assertEqual(content['count'], 0)
+
+    def test_resource_list_paging(self):
+        gen_res_one = resource.create_resource('CompositeResource', self.user, 'Resource 1',
```

Comment:
@sblack-usu Just curious, does this ```Resource 1``` parameter somehow indicate the resource should be created and displayed on page 2 or something? It looks like you are doing the right thing, but I have a difficult time to figure out how paging is done looking at this PR.

Reply:
It's just one of four resources I created.  Below, I set the number of resources per page to 1 and the page requested to 2 so I can check for the next/previous urls. (With 4 resources and 1 resource per page, there are 4 pages).